### PR TITLE
Cf 09242019

### PR DESCRIPTION
**Pull request by cforse**
I edited the Iphone 11 Pro product page

### CODE REVIEW

**Comment by cforse** [comment]
Here is my comment

File: README.md [hunk 1]
```diff
@@ -8,4 +8,9 @@ Links to different products and info
 Product image and description. Maybe add aside with customer endorsement.
 ## Footer Section
 Fake store address, social media links, etc.
-This is my line --Brian Murdock
\ No newline at end of file
+This is my line --Cameron Forsey
```
Comment (cforse) [inline README.md:10]: This is my comment that I am leaving 6:56

File: README.md [hunk 2]
```diff
@@ -8,4 +8,9 @@ Links to different products and info
 Product image and description. Maybe add aside with customer endorsement.
 ## Footer Section
 Fake store address, social media links, etc.
-This is my line --Brian Murdock
```
Comment (cdixon16-maker) [inline README.md]: This is a test

File: index.html [hunk 3]
```diff
@@ -28,6 +28,9 @@ <h1>Sample Web Store</h1>
                 <li>
                     <a href="#">Product 4</a>
                 </li>
+                <li>
+                    <a href="/Users/cameronforsey/Documents/Helio/Web-Store/Iphone11Pro.html">Iphone 11 Pro</a>
+                </li>
                 <li>
```
Comment (pilgrimk) [inline index.html]: I've added a comment here, -Kevin

**Pull request merged**